dbt_project.yml

```diff
@@ -34,10 +36,9 @@ vars:
     contact_link: "{{ ref('stg_lever__contact_link') }}"
     resume: "{{ ref('stg_lever__resume') }}"
 
-    lever_using_requisitions: true
```

Comment:
This was nested, but I don't think it should have been. The fact that it is called with `var('lever_using_requisitions', True)` suggests to me it _should_ be top level. The way it is now, nested under `lever:`, I think it would have to be called with `{{var('lever')["lever_using_requisitions"]}}` - but I could be mistaken, parsing yml is not my forte